packages/api-client/src/libs/create-client.ts

```diff
@@ -110,6 +111,11 @@ export const createApiClient = ({
   // Create the sidebar state
   const sidebarState = createSidebarState({ layout })
 
+  // Create the plugin manager
+  const pluginManager = createPluginManager({
```

Comment:
do you want to pass in the pluginManager from references? Only create this one if it doesn't exist like we do for store? Or does it not matter?